From a0cd81e4312733a1d00260690fe3f23caea23d9a Mon Sep 17 00:00:00 2001
From: Daniel Nachun <daniel.nachun@gmail.com>
Date: Wed, 23 Oct 2024 11:39:42 -0700
Subject: [PATCH 2/2] fix headers

---
 src/liblwgeom/lookup3.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/liblwgeom/lookup3.c b/src/liblwgeom/lookup3.c
index 67f4da2..d08a461 100644
--- a/src/liblwgeom/lookup3.c
+++ b/src/liblwgeom/lookup3.c
@@ -38,7 +38,9 @@ on 1 byte), but shoehorning those bytes into integers efficiently is messy.
 #include <stdio.h>      /* defines printf for tests */
 #include <time.h>       /* defines time_t for timings in the test */
 #include <stdint.h>     /* defines uint32_t etc */
+#ifndef WIN32
 #include <sys/param.h>  /* attempt to define endianness */
+#endif
 #ifdef linux
 # include <endian.h>    /* attempt to define endianness */
 #endif
-- 
2.45.2

